provider/k8s/controller_deployment.go

```diff
@@ -154,9 +156,11 @@ func (c *DeployController) SyncPDB(d *apps.Deployment, remove bool) error {
 		remove = true
 	}
 
-	minAvailable := &intstr.IntOrString{
+	c.Provider.PdbDefaultMinAvailablePercentage = common.CoalesceString(os.Getenv("PDB_DEFAULT_MIN_AVAILABLE_PERCENTAGE"),"50")
+
+	pdb_default_min_available_percentage := &intstr.IntOrString{
```

Comment:
in golang we don't use snakecase, fix it please.

Reply:
Corrected this, will this in my mind.